platform/android/sdk/src/com/ansca/corona/SystemMonitor.java

```diff
@@ -139,9 +139,14 @@ public boolean isScreenOff() {
 	public boolean isScreenLocked() {
 		android.app.KeyguardManager keyguardManager;
 		keyguardManager = (android.app.KeyguardManager)fContext.getSystemService(android.content.Context.KEYGUARD_SERVICE);
-		return keyguardManager.isDeviceLocked();
+		//Use new/updated method if supported
+		if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
LOLLIPOP_MR1 is equivalent to Android 5.1 (see https://apilevels.com/) 
We are experiencing a very similar sounding bug on Android 5.0.2: https://github.com/coronalabs/corona/issues/437
when building with:
https://github.com/coronalabs/corona/releases/tag/3677

Is it possible this line needs to be changed to 

```Java
if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP) {
```

@scottrules44 @Shchvova

Reply:
I already responded your comment but Lollipop does not support isDeviceLocked api so lowing the version would not help